packages/@okta/vuepress-site/docs/guides/oie-upgrade-mfa-enroll-policy/index.md

```diff
@@ -0,0 +1,178 @@
+
+## Get MFA Enrollment Policies
+
+To parse a response from the `GET /api/v1/policies?type=MFA_ENROLL` request, you must determine if the returned MFA Enrollment Policy contains either `authenticators` or `factors` in the [settings schema](/docs/reference/api/policy/#policy-settings-data-2).
+
+> **Note:** Another indicator for an authenticator MFA Enrollment Policy is that the [settings schema](/docs/reference/api/policy/#policy-settings-data-2) contains a `type` property that is set to the `AUTHENTICATORS` value.
+
+If the returned policy uses `authenticators`, you need to loop through the list of [Policy Authenticator objects](/docs/reference/api/policy/#policy-authenticator-object) and use the `key` property to identify the authenticator. See the [Authenticator Administration API](/docs/reference/api/authenticators-admin/) for more details on the available authenticators in your org.
+
+If the returned policy uses `factors`, you need to loop through every type of [Policy Factor Configuration object](/docs/reference/api/policy/#policy-factors-configuration-object), as you've done previously for your app.
+
+Example of an MFA Enrollment Policy response `settings` snippet with authenticators:
+
+```json
+"settings": {
+  "type": "AUTHENTICATORS",
+  "authenticators": [
+    {
+      "key": "security_question",
+      "enroll": {
+        "self": "OPTIONAL"
+      }
+    },
+    {
+      "key": "phone_number",
+      "enroll": {
+        "self": "REQUIRED"
+      }
+    }
+  ]
+}
+```
+
+Example of an MFA Enrollment Policy response `settings` snippet with factors:
+
+```json
+"settings": {
+  "factors": {
+    "okta_question": {
+      "enroll": {
+        "self": "OPTIONAL"
+      },
+      "consent": {
+        "type": "NONE"
+      }
+    },
+    "okta_sms": {
+      "enroll": {
+        "self": "REQUIRED"
+      },
+      "consent": {
+        "type": "NONE"
+      }
+    }
+  }
+}
+```
+
+## Create an MFA Enrollment Policy
+
+To create an MFA Enrollment Policy through the [Policy API](/docs/reference/api/policy/), you need to provide the `settings` schema with either the list of [Policy Authenticator objects](/docs/reference/api/policy/#policy-authenticator-object) or [Policy Factors Configuration objects](/docs/reference/api/policy/#policy-factors-configuration-object) in the `POST /api/v1/policies` request parameters body.
+
+For a new MFA Enrollment Policy in Identity Engine, use the [Create a Policy](/docs/reference/api/policy/#create-a-policy) API operation. Set the list of authenticators for the policy by using the [Authenticators Administration API](/docs/reference/api/authenticators-admin/) to list the available authenticators in your org.
+
+You can also create a new MFA Enrollment Policy with factors instead of authenticators to support legacy systems or workflows. Set the policy `settings` to the factors schema with [Policy Factor Configuration objects](/docs/reference/api/policy/#policy-factors-configuration-object).
```

Comment:
```suggestion
You can also create a new MFA enrollment policy with factors instead of authenticators to support legacy systems or workflows. Set the policy `settings` to the factors schema with [Policy Factor Configuration objects](/docs/reference/api/policy/#policy-factors-configuration-object).
```